### DISCUSSION TITLE
How to exclude archived repos?

### DISCUSSION THREAD

**Discussion by AdityaKane2001** [lowlighter/metrics#466] [Q&A] [answered]
How to exclude archived repos from metrics calculation?

**Comment by lowlighter** [comment, marked as answer]
Hello!

This is not possible currently, as GitHub GraphQL API does not allow filtering repositories by using `isArchived` field:
![image](https://user-images.githubusercontent.com/22963968/129188929-79a8b0c5-1bcf-4330-950d-316995453d6f.png)

If this option become available in the future, we could add it into metrics 👍
